tests/unit/src/main/scala/scala/meta/internal/metals/debug/DummyServer.scala

```diff
@@ -0,0 +1,31 @@
+package scala.meta.internal.metals.debug
+import org.eclipse.lsp4j.jsonrpc.MessageConsumer
+import org.eclipse.lsp4j.jsonrpc.messages.Message
+import org.eclipse.lsp4j.jsonrpc.messages.ResponseMessage
+import scala.collection.mutable
+
+final class DummyServer(
```

Comment:
For consistency with `TestingServer` and `TestingClient`?
```suggestion
final class TestingDebugServer(
```

Reply:
good catch. In fact, we should use the Scala `Long` here